lib/MusicBrainz/Server/Controller/Admin.pm

```diff
@@ -134,7 +134,7 @@ sub delete_user : Path('/admin/user/delete') Args(1) RequireAuth HiddenOnSlaves
     }
 }
 
-sub edit_banner : Path('/admin/banner/edit') Args(0) RequireAuth(banner_editor) {
+sub edit_banner : Path('/admin/banner/edit') Args(0) RequireAuth(banner_editor) CSRFToken {
```

Comment:
I was wondering why some subroutines had to separately check `validate_csrf_token` and found out that `form_posted_and_valid` just call  `form_submitted_and_valid`. That is explained in the previous commit. :+1:

Reply:
Yeah, e.g. in `delete_user` above I manually call `validate_csrf_token`. Because apparently we only check that it's a POST request there - there's no Form to validate against.